src/adic_space.lean

```diff
@@ -17,54 +20,55 @@ universe u
 
 open nat function
 open topological_space
-open Spa
+open spa
 
 namespace sheaf_of_topological_rings
 
-instance topological_space {X : Type*} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
+instance topological_space {X : Type u} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
```

Comment:
Why those changes from `Type*` to `Type u`? Is it faster to elaborate?

Reply:
I don't know anymore... but I guess it doesn't hurt.